unblob/handlers/archive/arj.py

```diff
@@ -28,7 +32,12 @@ class ARJHandler(StructHandler):
         strings:
             $arj_magic = { 60 EA [5] 0? [2] 0? }
         condition:
-            $arj_magic
+            $arj_magic and
+            // CPP/7zip/Archive/ArjHandler.cpp IsArc_Arj()
+            // basic_header.size must be between 30 and 2600
+            uint16(@arj_magic+2) > 30 and uint16(@arj_magic+2) < 2600 and
+            // arj_header.first_hdr_size is less then basic_header.size
+            uint16(@arj_magic+2) > uint8(@arj_magic+3)
```

Comment:
Shouldn't it be `uint16(@arj_magic+2) > uint8(@arj_magic+4)` ?

If `basic_header` is made of two uint16, that means we need to skip 4 bytes to reach `first_hdr_size`.

Reply:
Moved to chunk logic (see #312 )